cms-lite/src/main/resources/webapp/partials/widgets/string-resource-modal.html

```diff
@@ -32,7 +32,7 @@ <h4 class="modal-title" id="stringResourceLabel">{{msg('cmslite.header.resource'
             <div class="modal-footer">
                 <!-- read mode -->
                 <a ui-if="mode == 'read'"  class="btn btn-default pull-left"
-                   href="../cmsliteapi/string/{{select.language}}/{{select.name}}/"
+                   href="{{ '../cmsliteapi/resource/string/' + select.language + '/' + select.name }}"
```

Comment:
This button should lead us directly to the resource content. I believe after changing this link to contain resource, it will display us the String/Stream Content entity associated with this content. Can you verify and fix if necessary?